xray/src/bin/merge_xray_quadtrees.rs

```diff
@@ -20,8 +27,179 @@ struct CommandlineArguments {
     input_directories: Vec<PathBuf>,
 }
 
+fn copy_images(input_directory: &Path, output_directory: &Path) {
+    if input_directory == output_directory {
+        return;
+    }
+    globwalk::GlobWalkerBuilder::new(
+        input_directory,
+        &format!("*.{}", xray::IMAGE_FILE_EXTENSION),
+    )
+    .build()
+    .expect("Failed to build GlobWalker")
+    .filter_map(Result::ok)
```

Comment:
I'm filtering out errors here (i'm not sure when we can get errors, tbh, i coulnd find it in the documentation of glob_walker). Do you think that instead of filtering we should panic on error?

Reply:
No, I think it's ok as it is.